tensorflow/tools/api/golden/tensorflow.nn.rnn_cell.-s-r-u-cell.pbtxt

```diff
@@ -0,0 +1,152 @@
+path: "tensorflow.nn.rnn_cell.SRUCell"
```

Comment:
Please revert this change. It should have gone through its own API review. I will revert the culprit change to be submitted with an API review.